### PR TITLE
Only index collections of type NUL Collection in Elasticsearch Image documents

### PR DESCRIPTION
**Pull request by kdid**
* Only indexes collection information on the Image document in Elasticsearch if the collection is of the type that matches the `Settings.nul_collection_type`

### CODE REVIEW

File: app/services/common_indexers/image.rb [hunk 1]
```diff
@@ -73,6 +73,10 @@ def fields
 
     private
 
+      def nul_collections
+        member_of_collections.select { |c| c.collection_type_gid == Settings.nul_collection_type }.map { |c| { id: c.id, title: c.title.to_a } }
```
Comment (bmquinn) [inline app/services/common_indexers/image.rb]:
```suggestion
        member_of_collections.select { |c| Hyrax::CollectionType.find_by(machine_id: Hyrax::CollectionType::USER_COLLECTION_MACHINE_ID).collections.include?(c) }.map { |c| { id: c.id, title: c.title.to_a } }
```

Comment (kdid) [reply app/services/common_indexers/image.rb]: I don't think that will work. We're trying to only add collection information if it's an "NUL Collection" (or "NUL Collections" on staging) type collection - which is a manually created collection type. - it's not a user collection type.

Comment (bmquinn) [reply app/services/common_indexers/image.rb]: Sorry I misunderstood the feature. I think you could do something like `Hyrax::CollectionType.find_by(title: "NUL Collection")` or the like instead.

**Comment by kdid** [comment]
@bmquinn I don't think I want to go that route. I started to go that route on Glaze only to realize they named staging and production differently.  😢

**Pull request merged**